packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -150,3 +151,25 @@ export const getUnfurlMetadata = (
   const message = getChatMessageById(state, chatId, messageId)
   return message?.unfurlMetadata
 }
+
+export const getCanMessageUser = createSelector(
+  [
+    getBlockers,
+    getBlockees,
+    getUserChatPermissions,
+    (_: CommonState, userId: ID) => userId
```

Comment:
Might be missing something, but wondering why this selector takes state and then doesn't use it? Can it be rewritten to not accept the state? Or does it always have to accept it as the first argument?

Reply:
Will double check but I think the state has to be the first arg.